my_compassion/templates/my2_assets.xml

```diff
@@ -10,6 +10,7 @@
             <script type="text/javascript" src="/my_compassion/static/src/js/toast_service.js" />
             <script type="text/javascript" src="/my_compassion/static/src/js/my2_child_letters.js" />
             <script type="text/javascript" src="/my_compassion/static/src/js/my2_donation_form.js" />
+            <script type="text/javascript" src="/my_compassion/static/src/js/my2_prevent_modal_resize_error.js" />
```

Comment:
The current name, `my2_prevent_modal_resize_error.js`, describes the problem it solves. For long-term maintainability, the file name should describe the role or function of the code inside, not the specific bug/goal.